application/common/components/passwordStore/Ldap.php

```diff
@@ -312,9 +313,9 @@ protected function updatePassword($user, $password)
     }
 
     /**
-     * @param \Adldap\Models\Entry $user
+     * @param \LdapRecord\Models\OpenLDAP\Entry $user
      */
-    protected function removeAttributesAfterNewPassword($user)
+    protected function removeAttributesAfterNewPassword(Entry $user): void
```

Comment:
I like the stronger typing. I don't like using just Entry.

Reply:
It's a bit risky on old code, but possibly worth it.